.github/workflows/e2e_pr_full.yml

```diff
@@ -0,0 +1,99 @@
+name: Monitor E2E Test Suite (full)
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+  e2e-tests:
+    timeout-minutes: 60
```

Comment:
The timeout value is the same as for the “Monitor Cron e2e Tests” — might be a bit high for the accelerated runtime of the E2E tests. We could consider decreasing the values for both.